Gems/PhysX/Code/Source/EditorRigidBodyComponent.cpp

```diff
@@ -317,6 +351,21 @@ namespace PhysX
         }
     }
 
+    const bool EditorRigidBodyComponent::IsSceneCCDDisabled() const
```

Comment:
I agree but I can't seem to use a negative with the Attributes in Reflect(), if I use !IsSceneCCDEnabled then it won't change the visibility

Reply:
Hmm I see... can you maybe have a public function that's called `IsSceneCcdEnabled`, and then make another private one which is called `IsSceneCcdDisabled` which just calls `!IsSceneCcdEnabled()`. This makes other calling code simple but handles the annoying internal case with the EditContext? It might also be possible to use a lambda instead maybe? So something like....

```
[this]
{
    return !IsSceneCcdEnabled();
}
```